indigo_api/models/works.py

```diff
@@ -400,97 +400,41 @@ def has_consolidation(self):
-                fully_commenced = not bool(self.all_uncommenced_provision_ids(scoped_date))
-
-            description['type'] = 'single'
-            description['in_full'] = fully_commenced
-
-            if commencement.commencing_work:
-                description['commenced_by'] = {
-                    'work': commencement.commencing_work,
-                    'title': commencement.commencing_work.numbered_title() or commencement.commencing_work.title}
-
-            if commencement.note:
-                description['note'] = _('Note: %(commencement_note)s') % {'commencement_note': commencement.note}
-
-            commencement_date = commencement.date
-            if commencement_date:
-                description['date'] = commencement_date
-                if friendly_date:
-                    commencement_date = date_format(commencement_date, 'j E Y')
-
-                # with a date and a commencing work
-                if description['commenced_by']:
-                    if fully_commenced:
-                        description['description'] = _('Commenced in full on %(date)s by') % {'date': commencement_date}
-                    else:
-                        description['description'] = _('Commenced in part on %(date)s by') % {'date': commencement_date}
-
-                # with a date, without a commencing work
-                elif fully_commenced:
-                    description['description'] = _('Commenced in full on %(date)s') % {'date': commencement_date}
-                else:
-                    description['description'] = _('Commenced in part on %(date)s') % {'date': commencement_date}
-
-            # without a date, with a commencing work
-            elif description['commenced_by']:
-                if fully_commenced:
-                    description['description'] = _('Commenced in full by')
-                else:
-                    description['description'] = _('Commenced in part by')
-
-            # without a date or a commencing work
-            elif fully_commenced:
-                description['description'] = _('Commenced in full')
-            else:
-                description['description'] = _('Commenced in part')
+            return TimelineCommencementEvent(subtype='uncommenced', description=_('Not commenced'))
 
-        return description
+        # single commencement
+        commencement = commencements[0] if commencements else self.commencements.first()
+        return describe_single_commencement(commencement, with_date=True, friendly_date=friendly_date)
 
     def commencement_description_internal(self):
         return self.commencement_description(friendly_date=False)
 
     def commencement_description_external(self):
         return self.commencement_description()
 
+    def get_timeline(self):
+        from indigo_api.models.timeline import get_timeline
+        return get_timeline(self)
+
+    def get_timeline_as_dicts(self):
```

Comment:
Minor: since this is serializing them, normally this would be done by the place that cares about serializing, since this class doesn't really.